sql/mysqld.h

```diff
@@ -374,6 +374,13 @@ inline double my_timer_to_microseconds(ulonglong when) {
   ret /= (double)(my_timer.frequency);
   return ret;
 }
+/* Convert native timer units in a ulonglong into microseconds in a ulonglong */
+inline ulonglong my_timer_to_microseconds_ulonglong(ulonglong when) {
+  ulonglong ret = (ulonglong)(when);
```

Comment:
when is already of type `ulonglong`
```suggestion
  ulonglong ret = when;
```